### DISCUSSION TITLE
Create3 does not connect to the wifi network

### DISCUSSION THREAD

**Discussion by halimimaha** [iRobotEducation/create3_docs#540] [Questions]
The create3 does not want to connect to the wifi, on the other hand the turtlebot4 connects to the wifi, you will find the details in the following picture : 

![Screenshot from 2024-02-07 17-40-10](https://github.com/iRobotEducation/create3_docs/assets/126586495/fdd07379-6e18-4727-9f20-ad31835a1205)

Thanks in advance !

**Comment by slowrunner** [comment]
@halimimaha, suggest adding info about:
- your bluetooth/USB switch position
- your Create3 configuration file
- the exact steps you followed, in detail, to connect the Create3 to WiFi
- Is the Create3 networked to the TB4 controller/processor via USB0?

**Comment by halimimaha** [comment]
Here are the additional information : 

1- Im not sure for this answer but my usb is connect to usb1 port 

2-  Create3 configuration  : 

![Screenshot from 2024-02-13 10-32-14](https://github.com/iRobotEducation/create3_docs/assets/126586495/cea25690-da12-4589-b11e-ac20266c401e)

3- The exact steps that i followed to connect the Create3 to WiFi : 

![Screenshot from 2024-02-13 10-33-52](https://github.com/iRobotEducation/create3_docs/assets/126586495/953526bd-9934-4d0b-ac92-d2a882347b43)

4- Yes i think, there is a USB cable connect tb4 to Create3 and in the pic below we can see usb0 :

![Screenshot from 2024-02-13 10-35-42](https://github.com/iRobotEducation/create3_docs/assets/126586495/bdb0603f-5e02-4b44-8a05-abcea948a68d)
